src/rules.scala

```diff
@@ -105,7 +106,10 @@ object Plugin extends sbt.Plugin {
         compile in Compile <<= compile in Compile dependsOn(
           packageT in Compile in p),
         localProjects += LibraryProject((projectLayout in p).value),
-        localProjects <++= localProjects in p
+        localProjects := {
```

Comment:
I hate using `:=` when it makes code more verbose, `localProjects <++= (localProjects in p) map (_.distinctLibs)`